src/main/java/org/opentripplanner/model/TimetableSnapshot.java

```diff
@@ -424,7 +424,7 @@ public int compare(Timetable t1, Timetable t2) {
    * Class to use as key in HashMap containing feed id, trip id and service date
    * TODO shouldn't this be a static class?
    */
-  protected static class TripIdAndServiceDate {
+  public static class TripIdAndServiceDate {
```

Comment:
Public inner classes are discouraged. Move it to a top level class in the same package.